### PR TITLE
2D

### PR DESCRIPTION
**Pull request by je-santos**

### CODE REVIEW

File: src/pysimfrac/src/methods/to_2D.py
Comment (hymanjd) [inline src/pysimfrac/src/methods/to_2D.py:1]: I believe mean_aperture is already on the object and I'm not sure why the default is 28. Can you explain that default?

File: src/pysimfrac/src/general/simFrac.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 """
 SimFrac object class. 
 """
-import os, sys
```
Comment (hymanjd) [inline src/pysimfrac/src/general/simFrac.py:10]: Why was this line removed?

**Review by hymanjd — CHANGES_REQUESTED** [review]
Overall looks good. A few comments for clarification

File: src/pysimfrac/src/methods/to_2D.py
Comment (agnemarcato) [reply src/pysimfrac/src/methods/to_2D.py:1]: I see your point. I can modify this function and the example. I can avoid having this argument in the generate_2D function and I can just define the parameter in the example (as usual). If you accept this PR I can modify it right after.

Comment (hymanjd) [reply src/pysimfrac/src/methods/to_2D.py:1]: Sounds good. Will do now.

**Review by hymanjd — APPROVED** [review]

File: src/pysimfrac/src/general/simFrac.py (hunk 1 above)
Comment (agnemarcato) [reply src/pysimfrac/src/general/simFrac.py:10]: Javier told me and I checked that it's not used in the script

Comment (hymanjd) [reply src/pysimfrac/src/general/simFrac.py:10]: We use os and sys throughout the class for file manipulation and error checking. Unless we have run all the examples and confirmed those work without these, I vote to leave it in. Also, it's not an essential change from the new 2D source code. So, this would be a pull request with multiple facets, which is not best practices.

**Comment by agnemarcato** [comment]
Thank you! As soon as the code is merged I update the function and the example for the generation of 2D fractures with a new PR.

**Comment by hymanjd** [comment]
Done.

From: agnemarcato ***@***.***>
Reply-To: lanl/pySimFrac ***@***.***>
Date: Monday, November 13, 2023 at 10:16 AM
To: lanl/pySimFrac ***@***.***>
Cc: Jeffrey Hyman ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] Re: [lanl/pySimFrac] 2D (PR #3)


Thank you! As soon as the code is merged I update the function and the example for the generation of 2D fractures with a new PR.
